ui/StatusQ/sandbox/pages/StatusCommunityCardPage.qml

```diff
@@ -23,7 +23,7 @@ GridLayout {
     Repeater {
         model: Models.curatedCommunitiesModel
         delegate: StatusCommunityCard {
-            locale: "en"
+            locale: Qt.locale("en")
```

Comment:
These must go too, it won't work, because StatusCommunityCard has:

```
 res = Number(number).toLocaleString(Qt.locale(root.locale), 'f', 0)
```

It's expecting a string, not a Locale object

Reply:
That changed [here](https://github.com/status-im/status-desktop/pull/8911/files#diff-c2deee7788263b86efa453797db3198800dbc44b84f65710c1089f44c56ccd8a), it's now using the `root.locale` directly
`
        function numberFormat(number) {
            var res = number
            const million = 1000000
            const ks = 1000
            if(number > million) {
                res = number / million
                res = Number(number / million).toLocaleString(root.locale, 'f', 1) + 'M'
            }
            else if(number > ks) {
                res = number / ks
                res = Number(number / ks).toLocaleString(root.locale, 'f', 1) + 'K'
            }
            else
                res = Number(number).toLocaleString(root.locale, 'f', 0)
            return res
        }
`

Reply:
Ah ok then, good